lib/nerves_bootstrap/aliases.ex

```diff
@@ -47,7 +48,9 @@ defmodule Nerves.Bootstrap.Aliases do
   def add_target_aliases(aliases) do
     aliases
     |> prepend("deps.loadpaths", "nerves.loadpaths")
+    |> prepend("deps.loadpaths", "nerves.bootstrap")
     |> prepend("deps.compile", "nerves.loadpaths")
+    |> prepend("deps.compile", "nerves.bootstrap")
```

Comment:
Yes, we need `nerves.bootstrap` to run in each of these tasks to ensure we're loaded into the tool, especially because each of these tasks could be run independently. But including this alias is a noop because Mix tasks only run once, regardless of how many times the task is called.

<details>
<summary>Here's some output showing that `nerves.bootstrap` is called only once even though `deps.compile` and `deps.get` were both called</summary>

```
> reduced √ % MIX_DEBUG=1 NERVES_DEBUG=1 MIX_TARGET=rpi0 mix do deps.get, deps.compile                                                                                                                                      merge-bootstrap
-> Running mix loadconfig (inside Reduced.MixProject)
<- Ran mix loadconfig in 11ms
-> Running mix do deps.get, deps.compile (inside Reduced.MixProject)
-> Running mix deps.get (inside Reduced.MixProject)
-> Running mix archive.check (inside Reduced.MixProject)
<- Ran mix archive.check in 0ms
Resolving Hex dependencies...
Dependency resolution completed:
Unchanged:
  beam_notify 1.0.0
  castore 0.1.17
  circular_buffer 0.4.1
  elixir_make 0.6.3
  gen_state_machine 3.0.0
  mdns_lite 0.8.5
  muontrap 1.0.0
  nerves_motd 0.1.7
  nerves_pack 0.7.0
  nerves_runtime 0.11.10
  nerves_ssh 0.3.0
  nerves_system_bbb 2.13.4
  nerves_system_br 1.18.6
  nerves_system_osd32mp1 0.9.4
  nerves_system_rpi 1.18.4
  nerves_system_rpi0 1.18.4
  nerves_system_rpi2 1.18.4
  nerves_system_rpi3 1.18.4
  nerves_system_rpi3a 1.18.4
  nerves_system_rpi4 1.18.4
  nerves_system_x86_64 1.18.4
  nerves_time 0.4.5
  nerves_toolchain_aarch64_nerves_linux_gnu 1.5.0
  nerves_toolchain_armv6_nerves_linux_gnueabihf 1.5.0
  nerves_toolchain_armv7_nerves_linux_gnueabihf 1.5.0
  nerves_toolchain_ctng 1.8.5
  nerves_toolchain_x86_64_nerves_linux_musl 1.5.0
  one_dhcpd 2.0.0
  property_table 0.2.0
  ring_logger 0.8.4
  shoehorn 0.9.1
  ssh_subsystem_fwup 0.6.0
  system_registry 0.8.2
  toolshed 0.2.26
  uboot_env 1.0.0
  vintage_net 0.12.0
  vintage_net_direct 0.10.6
  vintage_net_ethernet 0.11.0
  vintage_net_wifi 0.11.0
<- Ran mix deps.get in 1117ms
-> Running mix nerves.bootstrap (inside Reduced.MixProject)
|nerves_boostrap| load nerves START
-> Running mix deps.precompile (inside Reduced.MixProject)
<- Ran mix deps.precompile in 10ms
==> elixir_make
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix compile.yecc in 2ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix compile.elixir in 7ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside ElixirMake.Mixfile)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 24ms
<- Ran mix compile in 27ms
==> castore
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CAStore.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 22ms
<- Ran mix compile in 22ms
==> nerves
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.elixir_make in 15ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.elixir in 4ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 65ms
<- Ran mix compile in 65ms
==> reduced
|nerves_boostrap| load nerves END
<- Ran mix nerves.bootstrap in 456ms
-> Running mix nerves.deps.get (inside Reduced.MixProject)
|nerves| Nerves.Deps.Get Start


Nerves environment
  MIX_TARGET:   rpi0
  MIX_ENV:      dev

|nerves| Env Start

|nerves| Env End

-> Running mix nerves.artifact.get (inside Reduced.MixProject)
Checking for prebuilt Nerves artifacts...
  Found nerves_system_rpi0 in cache
    /Users/jonjon/.nerves/artifacts/nerves_system_rpi0-portable-1.18.4
  Found nerves_toolchain_armv6_nerves_linux_gnueabihf in cache
    /Users/jonjon/.nerves/artifacts/nerves_toolchain_armv6_nerves_linux_gnueabihf-darwin_arm-1.5.0
<- Ran mix nerves.artifact.get in 9ms
|nerves| Nerves.Deps.Get End

<- Ran mix nerves.deps.get in 719ms
-> Running mix nerves.loadpaths (inside Reduced.MixProject)
|nerves| Loadpaths Start

-> Running mix nerves.precompile --no-loadpaths (inside Reduced.MixProject)
|nerves| Precompile Start

|nerves| Precompile End

<- Ran mix nerves.precompile in 0ms

Nerves environment
  MIX_TARGET:   rpi0
  MIX_ENV:      dev

-> Running mix nerves.env (inside Reduced.MixProject)
|nerves| Env Start

|nerves| Env End

<- Ran mix nerves.env in 0ms
|nerves| Environment Variable List
  target:     rpi0
  toolchain:  /Users/jonjon/.nerves/artifacts/nerves_toolchain_armv6_nerves_linux_gnueabihf-darwin_arm-1.5.0
  system:     /Users/jonjon/.nerves/artifacts/nerves_system_rpi0-portable-1.18.4
  app:        .

|nerves| Loadpaths End

<- Ran mix nerves.loadpaths in 85ms
-> Running mix deps.compile (inside Reduced.MixProject)
==> ssh_subsystem_fwup
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix loadpaths in 1ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SSHSubsystemFwup.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 26ms
<- Ran mix compile in 27ms
==> nerves_system_br
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.System.BR.Mixfile)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.System.BR.Mixfile)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.System.BR.Mixfile)
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.System.BR.Mixfile)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 2ms
<- Ran mix compile in 2ms
==> circular_buffer
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside CircularBuffer.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 30ms
<- Ran mix compile in 30ms
==> gen_state_machine
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside GenStateMachine.Mixfile)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 16ms
<- Ran mix compile in 16ms
==> uboot_env
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix compile.elixir in 3ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside UBootEnv.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 22ms
<- Ran mix compile in 22ms
==> ring_logger
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside RingLogger.MixProject)
<- Ran mix compile.app in 2ms
<- Ran mix compile.all in 21ms
<- Ran mix compile in 21ms
==> system_registry
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside SystemRegistry.Mixfile)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 14ms
<- Ran mix compile in 14ms
==> shoehorn
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix compile.elixir in 4ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Shoehorn.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 25ms
<- Ran mix compile in 26ms
==> nerves_runtime
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.elixir_make in 6ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.elixir in 4ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Runtime.MixProject)
<- Ran mix compile.app in 1ms
<- Ran mix compile.all in 31ms
<- Ran mix compile in 31ms
==> toolshed
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix compile.yecc in 1ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix compile.elixir in 3ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Toolshed.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 19ms
<- Ran mix compile in 20ms
==> nerves_motd
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix compile.elixir in 1ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesMOTD.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 34ms
<- Ran mix compile in 34ms
==> nerves_ssh
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSSH.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 30ms
<- Ran mix compile in 30ms
==> one_dhcpd
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.elixir_make in 6ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.elixir in 1ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside OneDHCPD.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 21ms
<- Ran mix compile in 22ms
==> muontrap
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.elixir_make in 9ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MuonTrap.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 29ms
<- Ran mix compile in 29ms
==> property_table
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside PropertyTable.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 47ms
<- Ran mix compile in 47ms
==> nerves_toolchain_ctng
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
Compiling 1 file (.ex)
<- Ran mix compile.elixir in 148ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside Nerves.Toolchain.Ctng.MixProject)
Generated nerves_toolchain_ctng app
<- Ran mix compile.app in 3ms
<- Ran mix compile.all in 154ms
<- Ran mix compile in 155ms
==> nerves_toolchain_armv6_nerves_linux_gnueabihf
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
-> Running mix compile.nerves_package --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
|nerves| Compile.NervesPackage start

|nerves| Compile.NervesPackage end

<- Ran mix compile.nerves_package in 0ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
<- Ran mix compile.elixir in 1ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesToolchainArmv6RpiLinuxGnueabihf.MixProject)
Generated nerves_toolchain_armv6_nerves_linux_gnueabihf app
<- Ran mix compile.app in 1ms
<- Ran mix compile.all in 7ms
<- Ran mix compile in 7ms
==> beam_notify
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.elixir_make in 9ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.yecc in 1ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.elixir in 5ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside BEAMNotify.MixProject)
<- Ran mix compile.app in 3ms
<- Ran mix compile.all in 49ms
<- Ran mix compile in 49ms
==> vintage_net
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix loadpaths in 3ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.elixir_make in 50ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.elixir in 11ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNet.MixProject)
<- Ran mix compile.app in 1ms
<- Ran mix compile.all in 147ms
<- Ran mix compile in 151ms
==> mdns_lite
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix compile.erlang in 10ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix compile.elixir in 4ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside MdnsLite.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 48ms
<- Ran mix compile in 48ms
==> vintage_net_direct
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetDirect.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 120ms
<- Ran mix compile in 121ms
==> vintage_net_ethernet
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix compile.elixir in 6ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetEthernet.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 55ms
<- Ran mix compile in 56ms
==> vintage_net_wifi
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.elixir_make in 51ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.yecc in 21ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.leex in 8ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.elixir in 12ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside VintageNetWiFi.MixProject)
<- Ran mix compile.app in 0ms
<- Ran mix compile.all in 154ms
<- Ran mix compile in 154ms
==> nerves_system_rpi0
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
<- Ran mix compile.elixir in 4ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
Generated nerves_system_rpi0 app
<- Ran mix compile.app in 0ms
-> Running mix compile.nerves_package --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesSystemRpi0.MixProject)
|nerves| Compile.NervesPackage start

|nerves| Compile.NervesPackage end

<- Ran mix compile.nerves_package in 0ms
<- Ran mix compile.all in 27ms
<- Ran mix compile in 28ms
==> nerves_time
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
-> Running mix compile.elixir_make --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.elixir_make in 9ms
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.yecc in 2ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.leex in 1ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.elixir in 5ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesTime.MixProject)
<- Ran mix compile.app in 5ms
<- Ran mix compile.all in 47ms
<- Ran mix compile in 47ms
==> nerves_pack
-> Running mix compile --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
-> Running mix loadpaths --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix loadpaths in 0ms
-> Running mix compile.all --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
-> Running mix compile.yecc --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix compile.yecc in 0ms
-> Running mix compile.leex --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix compile.leex in 0ms
-> Running mix compile.erlang --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix compile.erlang in 0ms
-> Running mix compile.elixir --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix compile.elixir in 2ms
-> Running mix compile.app --from-mix-deps-compile --no-archives-check --no-warnings-as-errors (inside NervesPack.MixProject)
<- Ran mix compile.app in 2ms
<- Ran mix compile.all in 12ms
<- Ran mix compile in 13ms
==> reduced
-> Running mix will_recompile (inside Reduced.MixProject)
<- Ran mix will_recompile in 0ms
<- Ran mix deps.compile in 1309ms
<- Ran mix do in 3693ms
```
</details>